### DISCUSSION TITLE
Use Heroku scheduler for cron jobs

### DISCUSSION THREAD

**Discussion by fynsta** [firefly-iii/firefly-iii#5820] [Firefly III] [answered]
Is there a way to use the heroku scheduler addon for cron jobs? 

Specifically, is there some kind of app entry point that I can have it call?

**Comment by JC5** [comment, marked as answer]
I don’t know how Heroko works exactly but this page has info on the cron job:

https://docs.firefly-iii.org/firefly-iii/advanced-installation/cron/

**Reply by fynsta** [reply]
Yeah I saw that page, but that basically just tells me to fire a curl request every day. I would want it to use some kind of entry point inside the project instead of an external API request.

This is how it theoretically works: https://devcenter.heroku.com/articles/scheduler#defining-tasks
For Rails I would define rake tasks, but for php there would have to be some script that the scheduler can call?

**Reply by JC5** [reply]
You mean the command mentioned on that page?

/usr/bin/php /var/www/html/artisan firefly-iii:cron

**Reply by fynsta** [reply]
Oh don't know how I overread that... Thank you!
